pepys_import/file/file_processor.py

```diff
@@ -175,7 +182,7 @@ def process_file(self, file, current_path, data_store, processed_ctr):
 
             # If all tests pass for all parsers, commit datafile
             # get current time without milliseconds
-            timestamp = str(datetime.utcnow())[:-7]
+            timestamp = sanitize_filename(str(datetime.utcnow())[:-7])
```

Comment:
Why do we need sanitize_filename?

Reply:
The file we generate contains characters that are illegal on MS-Windows:
```
Traceback (most recent call last):
  File "C:\Apps\Python37\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "C:\Apps\Python37\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\git\pepys-import\pepys_import\import.py", line 35, in <module>
    main(path=args.path, db=args.db)
  File "C:\git\pepys-import\pepys_import\import.py", line 22, in main
    processor.process(path, data_store, True)
  File "C:\git\pepys-import\pepys_import\file\file_processor.py", line 101, in process
    file, current_path, data_store, processed_ctr
  File "C:\git\pepys-import\pepys_import\file\file_processor.py", line 194, in process_file
    "w",
OSError: [Errno 22] Invalid argument: 'C:\\git\\pepys-import\\tests\\sample_data\\track_files\\output\\gpx_1_0_output_2020-03-06 08:57:11.log'
```

So, we need to either sanitize it, or change the underlying format of the timestamp string.

Reply:
Okay, I understand now. I think we won't need to sanitize it after #123 is solved. Because we will use `OUTPUT_PATH \ YYYY \ MM \ DD \ HH \ MM \ SS` to put our logs and files. So, I think we won't need timestamp strings in the filenames.